frontend/src/components/App/Home/index.tsx

```diff
@@ -20,18 +20,22 @@ import Event from '../../../lib/k8s/event';
 import { createRouteURL } from '../../../lib/router';
 import { useId } from '../../../lib/util';
 import { setConfig } from '../../../redux/configSlice';
+import { useTypedSelector } from '../../../redux/reducers/reducers';
 import { Link, PageGrid, SectionBox, SectionFilterHeader } from '../../common';
 import { ConfirmDialog } from '../../common';
+import ErrorBoundary from '../../common/ErrorBoundary/ErrorBoundary';
 import ResourceTable from '../../common/Resource/ResourceTable';
 import RecentClusters from './RecentClusters';
 
-function ContextMenu({ cluster }: { cluster: Cluster }) {
+function ContextMenu({ cluster }: { cluster: Cluster; active: boolean }) {
```

Comment:
active prop is not used inside the component

Reply:
done